backend/config/serialization/User.yaml

```
@@ -11,7 +11,7 @@ App\Entity\User:
             groups: ['user:read', 'user:read:collection'] # Include phone in lists for admin view? Maybe create user:read:admin?
         roles:
              # groups: ['user:read'] # Removed from default user view
-             groups: ['admin:read'] # Only readable by admin
+             groups: ['admin:read', 'user:read:collection'] # Only readable by admin
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 13

(indentation)